backend/clickhouse/migrations/000135_create_readonly_user.up.sql

```diff
@@ -0,0 +1,25 @@
+CREATE USER IF NOT EXISTS highlight_readonly IDENTIFIED BY '' SETTINGS readonly = 2;
```

Comment:
TIL 2 means settings can be changed but only for self

Reply:
👍 also updated this to `readonly = 1` with `highlight_readonly_role` as `CHANGEABLE_IN_READONLY` to limit changes to other settings